fragments/functions.sh

```diff
@@ -497,16 +497,21 @@ installAppWithPath() { # $1: path to app to install in $targetDir
 
         # copy app to /Applications
         printlog "Copy $appPath to $targetDir"
-        if ! ditto "$appPath" "$targetDir/$appName"; then
-            cleanupAndExit 7 "Error while copying" ERROR
+        copyAppOut=$(ditto -v "$appPath" "$targetDir/$appName" 2>&1)
+        copyAppStatus=$(echo $?)
+        deduplicatelogs "$copyAppOut"
+        printlog "Debugging enabled, App copy output was:\n$logoutput" DEBUG
+        if [[ $appVerifyStatus -ne 0 ]] ; then
```

Comment:
shouldn't this be checking `copyAppStatus`?

Reply:
Did I copy-paste and not correct it all? Have to look further into this, but i guess you are right.